modules/global/src/de/diedavids/cuba/ceuesr/entity/Commentable.java

```diff
@@ -0,0 +1,12 @@
+package de.diedavids.cuba.ceuesr.entity;
+
+import com.haulmont.cuba.core.entity.Entity;
+
+import java.io.Serializable;
+
+/**
+ * marks all entity instances that can be commented
+ * @param <T>
+ */
+public interface Commentable<T> extends Entity<T> {
```

Comment:
This is the marker interface for all entity classes that support Commenting. The interface is for two reasons there:

* better types in the code 
* determins the entity selection drop down